Add ConfirmationDialog component.

justincan opened the pull request:
- Copied over from snapweb.


Note: The code references vanilla styles. What should happen here?

vesarautiainen commented:
So you overwrote the Vanilla color values with some hardcoded ones? I think that's the best option for now and good that you left the Vanilla references in comments. 

But in longer term we need to come up with a solution how to start migrating toolkit to use Vanilla. Any ideas? How is is done in snapweb if this piece of code is taken from there? Could we use the same approach here?

justincan commented:
@vesarautiainen I think we should merge #1 and import vanilla into the framework.

Commit:
Import Snapweb styles into toolkit.

- Add Vanilla CSS toolkit
- Add sass processing step
- Import snapweb styles from snapweb project.
- Exporting vanilla based snapweb styles globally

Commit:
Add ConfirmationDialog component.

- Copied over from snapweb.
